app/Platform/Actions/ResetPlayerProgressAction.php

```diff
@@ -144,16 +153,25 @@ public function execute(User $user, ?int $achievementID = null, ?int $gameID = n
             $user->playerAchievements()
                 ->whereIn('achievement_id', $achievementIds)
                 ->delete();
+
+            // Track the game reset.
+            PlayerProgressReset::create([
+                'user_id' => $user->id,
+                'type' => PlayerProgressResetType::Game,
+                'type_id' => $gameID,
+            ]);
         } else {
+            // If we fall into this block, it's because a user's account is being deleted.
+
             // Delete all maintainer unlock records related to these player_achievement entities.
             AchievementMaintainerUnlock::query()
                 ->whereIn('player_achievement_id', function ($query) use ($user) {
                     $query->select('id')->from('player_achievements')->where('user_id', $user->id);
                 })
                 ->delete();
 
-            // fulfill deletion request
             $user->playerGames()->forceDelete();
+            $user->playerSessions()->delete();
```

Comment:
As best as I can tell, we seem to be holding on to player sessions for deleted users. I think we probably should not do this.